tests/unit/test_expression_tree/test_concatenations.py

```diff
@@ -312,25 +312,6 @@ def test_domain_error(self):
         with self.assertRaisesRegex(pybamm.DomainError, "domain cannot be empty"):
             pybamm.DomainConcatenation([a, b], None)
 
-    def test_numpy_concatenation_simplify(self):
-        a = pybamm.Variable("a")
-        b = pybamm.Variable("b")
-        c = pybamm.Variable("c")
-        # simplifying flattens the concatenations into a single concatenation
-        self.assertEqual(
-            pybamm.NumpyConcatenation(pybamm.NumpyConcatenation(a, b), c).simplify().id,
-            pybamm.NumpyConcatenation(a, b, c).id,
-        )
-        self.assertEqual(
-            pybamm.NumpyConcatenation(a, pybamm.NumpyConcatenation(b, c)).simplify().id,
-            pybamm.NumpyConcatenation(a, b, c).id,
-        )
-        # check it works when calling numpy_concatenation
-        self.assertEqual(
-            pybamm.numpy_concatenation(pybamm.numpy_concatenation(a, b), c).id,
-            pybamm.NumpyConcatenation(a, b, c).id,
-        )
```

Comment:
can we keep this part of the test?

Reply:
Ah yes I checked we can keep it, I'll add it back.